handover_api/utils.py

```diff
@@ -31,9 +32,10 @@ def send_draft(draft):
     message = generate_message(sender.email, receiver.email, subject, template_name, context)
     message.send()
 
+
 def get_accept_url(handover):
-    # TODO: lookup the accept url
-    return 'https://itlab-1.gcb.duke.edu/accept?token=' + str(handover.token)
+    return reverse('accept-process') + "?token=" + str(handover.token)
```

Comment:
Determining the absolute URL is probably outside the scope of this feature. Within the framework, we'd either need a request object or to manually configure the sites framework. We can just put the base URL into settings.py

For now, just prefix the relative path with the hard-coded base URL.